### PR TITLE
[DO NOT MERGE] #24 - Trying [Gitlab's API gem](https://github.com/NARKOZ/gitlab)

### PR DESCRIPTION
**Pull request by mdeloupy**

### CODE REVIEW

File: bin/checklister [hunk 1]
```diff
@@ -113,6 +114,13 @@ command :create do |c|
   end
 end
 
+desc "New issue"
+command :new_issue do |c|
+  checklist = Checklister::Parser.new("./examples/simple-checklist.md")
+  Gitlab.create_issue(469016, checklist.to_params[:title], description: checklist.to_params[:body])
```
Comment (mdeloupy) [inline bin/checklister]: This project's id : https://gitlab.com/mdeloupy/checklister-testing

File: bin/checklister [hunk 2]
```diff
@@ -113,6 +114,13 @@ command :create do |c|
...
+  puts "Go check if that worked !"
```
Comment (mdeloupy) [inline bin/checklister]: It did ! 
capture d ecran 2015-09-16 a 21 39 36

**Comment by benichu** [comment]
I applied some of your typo fixes, directly to my branch (https://github.com/benichu/checklister/commit/5eb1b8817c33f680edce47ae804573ec4bf40265)

Consider your branch, a playground branch that will not be merged to master...

File: bin/checklister (hunk 2 above)
Comment (benichu) [inline bin/checklister]: :+1:

**Pull request closed without merging**